util/namespace.js

```diff
@@ -3,10 +3,13 @@
 const fileIO = require('./fileIO');
 
 const generateNamespace = (namespaceName, interfaces) => `// graphql typescript definitions
+/* tslint:disable */
 
 declare namespace ${namespaceName} {
 ${interfaces}
 }
+/* tslint:enable */
+
 `;
```

Comment:
this new line is unnecessary b/c the backtick is on the line below the last line.

with this, it'll actually end in two lines.